spec/value_status_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'spec_helper'
+require 'ronin/recon/value_status'
+require 'ronin/recon/worker'
+
+describe Ronin::Recon::ValueStatus do
+  module TestWorkers
```

Comment:
Might want to name the module `TestValueStatus` to avoid conflicts with other specs. When you define a module/class inside of a `describe` they are actually defined globally.